ClearKeep/Scenes/Home/Modifiers/InCallModifier.swift

```diff
@@ -0,0 +1,284 @@
+//
+//  InCallModifier.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 19/07/2022.
+//
+
+import UIKit
+import SwiftUI
+import CommonUI
+import ChatSecure
+var callController: UIViewController?
+
+struct InCallModifier: ViewModifier {
+	@State var isInMinimizeMode: Bool = false
+	
+	@Environment(\.injected) private var injected: DIContainer
+	@Environment(\.presentationMode) private var presentationMode: Binding<PresentationMode>
+	@Binding var isInCall: Bool
+	@ObservedObject var callViewModel: CallViewModel
+	@State private var location: CGPoint = Position.bottomTrailling.location
+	@GestureState private var startLocation: CGPoint? = nil // 1
+	
+	var simpleDrag: some Gesture {
+		DragGesture()
+			.onChanged { value in
+				var newLocation = startLocation ?? location // 3
+				newLocation.x += value.translation.width
+				newLocation.y += value.translation.height
+				self.location = newLocation
+			}.updating($startLocation) { (_, startLocation, _) in
+				startLocation = startLocation ?? location // 2
+			}
+			.onEnded({ _ in
+				self.location = self.getPosition(position: self.location).location
+			})
+		
+	}
+	
+	func body(content: Content) -> some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Modifiers/InCallModifier.swift#L40"></span>Function body should span 50 lines or less excluding comments and whitespace: currently spans 79 lines<br />
<code>function_body_length</code> <code>InCallModifier.swift:40</code></td>
    </tr>
  </tbody>
</table>